website/docs/providers/notifier_provider.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: NotifierProvider
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme/CodeBlock";
+import todos from "./notifier_provider/todos";
+import todosConsumer from "!!raw-loader!/docs/providers/notifier_provider/todos/todos_consumer.dart";
+import { trimSnippet, AutoSnippet } from "../../src/components/CodeSnippet";
+
+[NotifierProvider] is a provider that is used to listen to and expose a
+[Notifier].  
+[NotifierProvider] along with [Notifier] is Riverpod's recommended solution
+for managing state which may change in reaction to a user interaction.
+
+It is typically used for:
+
+- exposing an **immutable** state which can change over time after reacting to
```

Comment:
This isn't true anymore. Notifier supports both with `ref.notifyListeners()`

Reply:
removed **immutable**